pkg/session/service.go

```diff
@@ -191,7 +178,7 @@ func (s *KeycardService) GenerateMnemonic(args *GenerateMnemonicRequest, reply *
 }
 
 type LoadMnemonicRequest struct {
-	Mnemonic   string `json:"mnemonic" validate:"required"`
+	Mnemonic   string `json:"mnemonic" validate:"required,mnemonic"`
```

Comment:
Here is the improvement. 
I just also moved validation-related things to a separate file, but no changes there.